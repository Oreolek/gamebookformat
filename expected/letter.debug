BEGIN DEBUG OUTPUT
Book title: Letter
Number of sections: 1

Turn to 1 to begin.
1 (start) -  LaTeX output (and this PDF output) from this book should be in letter size rather than the default A5 size (see command-line options used in letter.gamebook.options).
END DEBUG OUTPUT
