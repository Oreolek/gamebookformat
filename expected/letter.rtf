{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Letter
\b0\
\b Turn to 1 to begin.
\b0\
\
\b \qc {\*\bkmkstart s1}1{\*\bkmkend s1}
\b0\
\ql  LaTeX output (and this PDF output) from this book should be in letter size rather than the default A5 size (see command-line options used in letter.gamebook.options). \
\

}
