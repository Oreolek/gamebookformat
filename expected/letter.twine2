<tw-storydata name="Letter" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> LaTeX output (and this PDF output) from this book should be in letter size rather than the default A5 size (see command-line options used in letter.gamebook.options).</tw-passagedata>
</tw-storydata>
