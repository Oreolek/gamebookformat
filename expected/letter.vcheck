examples/letter.json: ending (death?) section found: start
