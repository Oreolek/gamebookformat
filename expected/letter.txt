    Letter



Turn to 1 to begin.
1
 LaTeX output (and this PDF output) from this book should be in letter size rather than the default A5 size (see command-line options used in letter.gamebook.options).

